03.JS Advanced/CurrencyRates/src/app.js

```diff
@@ -0,0 +1,46 @@
+const key = 'Hpabx6qRquhyWI2aNTxFrnorRAEwrA22ByIr7iOn'
+
+$(function () {
+    let date = new Date()
+    date.setDate(date.getDate() - 1);
+    $('input[type="date"]').attr('max', date.toLocaleDateString('en-ca'))
+})
+
+async function requestExchangeRates() {
```

Comment:
Can be split into 2: one for gathering inputs from UI and creating query string, second: for fetching api and returning results.
Fetch functions should be reusable.